### DISCUSSION TITLE
How many requests per second can serverpod handle?

### DISCUSSION THREAD

**Discussion by mufayed** [serverpod/serverpod#598] [Q&A] [answered]
Hi all, 

Could you answer the following question, Please?
How many requests per second can serverpod handle?
Also. If you could provide us with a benchmark for serverpod that would be great, We're really excited to use serverpod as a backend for our projects.

Thanks

**Comment by vlidholt** [comment]
We would love help with some benchmarks! 😅 It's on the roadmap. Would be great to have so that we could improve and optimize over time too.

**Reply by braj065** [reply]
@vlidholt do you suggest that one should run serverpod locally to capture the benchmarks. Or it is imperative to run it on aws ec2 containers to capture real time benchmark? And what should be the setup(total ram, available ram before running tests, cpu etc), tool to fire requests, capture metrics and other details.

**Reply by abdelaziz-mahdy** [reply, marked as answer]
I've initiated a discussion on comparing Serverpod to various other backend technologies. You can find this discussion here: https://github.com/serverpod/serverpod/discussions/1791.

I chose to create a separate discussion instead of posting in this thread to make it easier for others to find, as I initially had difficulty locating similar topics despite thorough searching.
